agents-manage-ui/src/features/agent/domain/serialize.ts

```diff
@@ -557,7 +590,7 @@ export function serializeAgentData(
 
   const result: FullAgentDefinition = {
     id: metadata?.id || generateId(),
-    name: metadata?.name || 'Untitled Agent',
+    name: metadata?.name ?? '',
```

Comment:
field is marked as required, I think it's weird to under the hood add default name
<img width="119" height="86" alt="image" src="https://github.com/user-attachments/assets/dbc35ed6-529f-40f7-868a-44ac0376b231" />